webpack/scenes/ContentViews/Details/Versions/ContentViewVersions.js

```diff
@@ -66,6 +68,14 @@ const ContentViewVersions = ({ cvId, details }) => {
     [dispatch],
   );
 
+  useEffect(() => () => {
```

Comment:
I expected to see the explicit `return` keyword here, but realized that eslint won't allow it because of https://eslint.org/docs/rules/arrow-body-style.  I think someone who knows Hooks should be able to grok what's going on if the `return` keyword is there, but the implicit return of arrow functions makes it confusing and obfuscates what's going on. For that reason I think we should disable that eslint rule here and use `return`.
```suggestion
  useEffect(() => { // eslint-disable-line arrow-body-style
    return () => {
```

Reply:
I suppose it is good to make the return implicit for posterity. 
May our ancestors me be appeased! 🥲